WordPressKitTests/TestCollector+Constants.swift

```diff
@@ -0,0 +1,7 @@
+import BuildkiteTestCollector
+import Foundation
+
+extension TestCollector {
+
+    static let apiURL = URL(string: "https://analytics-api.buildkite.com/v1/uploads")!
+}
```

Comment:
I opened https://github.com/buildkite/test-collector-swift/issues/36 suggesting to make this part of the library.

Reply:
Might make more sense to make stubs to just test for the `analytics-api.buildkite.com` host, rather than relying on a specific full URL that might change in the future (especially if they update it in `v2` or subsequent versions of their APIs)?

You could then test it using a convenience OHHTTPStubs syntax like `stub(!isHost(TestCollector.apiHost)) { … }` — see [example usage in my other comment above](https://github.com/wordpress-mobile/WordPressKit-iOS/pull/530/files/bccc87031632b4e5bfe012c0ce99055fd38de0b6#r941308493).